executor/runtime/types/types.go

```diff
@@ -28,6 +29,7 @@ const (
 	KvmEnabledParam              = "titusParameter.agent.kvmEnabled"
 	assignIPv6AddressParam       = "titusParameter.agent.assignIPv6Address"
 	serviceMeshEnabledParam      = "titusParameter.agent.service.serviceMesh.enabled"
+	serviceMeshContainerParam    = "titusParameter.agent.service.serviceMesh.container"
```

Comment:
Maybe `titusParameter.agent.service.serviceMesh.containerImage`?  I figure we should be as explicit as possible about what this is - container seems a bit too generic.